wiki/GetStarted.md

```diff
@@ -1,22 +1,52 @@
 ## Get Started
 
-### Quick start
-The command line `geomPipeline.py imprint your_geometry_file_path` is the starting point for users, just replace *your_geometry_file_path* by your geometry. 
-
-run `geomPipeline.py -h` for more options. 
+### Command line interface
+
+Geometry pipeline can be started in a terminal. `geomPipeline.py` takes 2 positional arguments, the first is the action, and the second is the input file path. For example,  `geomPipeline.py imprint your_geometry_file_path` ,  just replace *your_geometry_file_path* by your geometry. 
+
+Optional arguments for all pipelines, it is expected other pipelines also follow this convention.
+```
+  -h, --help            show this help message and exit
+  -o OUTPUT_FILE, --output-file OUTPUT_FILE
+                        output file name (without folder path)
+  --working-dir WORKING_DIR
+                        working folder path, by default the current working folder
+  --output-dir OUTPUT_DIR
+                        output folder path, by default, a subfolder in workingDir
+  --config              only generate config.json without run the pipeline
+
+  -nt THREAD_COUNT, --thread-count THREAD_COUNT
+                        number of thread to use, max = hardware core number
+
+  -v VERBOSITY, --verbosity VERBOSITY
+                        verbosity: for console or terminal: DEBUG, PROGRESS, INFO, WARNING, ERROR
+```
```

Comment:
I think it's usually better to tell people to use the '-h' option, rather than duplicating information in the source code and markdown files (because with two versions you have twice as much to maintain, and it is very easy to forget to update the separate documentation).

Reply:
yes, we can remove it later